src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -165,6 +165,7 @@ protected override ILspServices ConstructLspServices()
         // TelemetryService.SetDefaultSession and provides the correct
         // appinsights keys etc
```

Comment:
Is this comment in the right place? It's not clear to me what code it's referring to.

Reply:
It looks like it should just be removed